test/resources/selinux.resource

```diff
@@ -0,0 +1,66 @@
+*** Settings ***
+Documentation       Keywords for OSTree-based systems
+
+Resource            ostree-data.resource
+Library             Collections
+Library             selinux.py
+
+
+*** Keywords ***
+Validate SELinux With Backup
+    [Documentation]    Wrapper call for all SELinux checks with extra check inside backup folder
+    [Arguments]    ${backup_name}
+
+    ${result}=    Run Container Access Check On    "${BACKUP_STORAGE}/${backup_name}/resources/kubeadmin/kubeconfig"
```

Comment:
Can you explain the idea a bit more on why we are only checking one file? 
Should we check the entire backup tree here?

Reply:
I intended to make this better, now its updated to check all files in a directory.